src/testHelpers.ts

```diff
@@ -107,3 +134,48 @@ export function getInformationFromTestnode(): TestnodeInformation {
 
   throw new Error('nitro-testnode sequencer not found');
 }
+
+export async function createRollupHelper({
+  l3TokenBridgeDeployer,
+  batchPoster,
+  validators,
+  nativeToken = zeroAddress,
+  client,
+}: {
+  l3TokenBridgeDeployer: PrivateKeyAccountWithPrivateKey;
```

Comment:
This should be renamed to just `deployer`, the fact that we're using a the `l3TokenBridgeDeployer` is just one case

Reply:
good point